src/iptvsimple/data/Channel.cpp

```diff
@@ -198,10 +200,11 @@ void Channel::TryToAddPropertyAsHeader(const std::string& propertyName, const st
 
 void Channel::SetCatchupDays(int catchupDays)
 {
-  if (catchupDays != 0)
+  if (catchupDays > 0)
```

Comment:
When `catchupDays` == -1 (ignore), this doesn't do anything.

Reply:
Faceplam

Reply:
Hmm, this might be okay, but the default is set to 0 in Channel::Reset

Reply:
The default is fine. We will always set it for VOD type.